ansible/roles/developer_packages/tasks/main.yml

```diff
@@ -40,3 +40,11 @@
       - bash-completion
       - autojump
       - nano
+
+  - name: Install Python package(s)
```

Comment:
Indentation is wrong. YAML is very picky about that. Make sure ewach new task has 2 preceding spaces,and then always use 2 spaces for each further indentation.